shardingsphere-agent-bootstrap should be renamed to shardingsphere-agent-core?

## Question

**For English only**, other languages will not accept.

Before asking a question, make sure you have:

- Googled your question.
- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere/issues).
- Read documentation: [ShardingSphere Doc](https://shardingsphere.apache.org/document/current/en/overview).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

- Now execute `./mvnw clean install -Release -T1C -DskipTests -Djacoco.skip=true -Dcheckstyle.skip=true -Drat.skip=true -Dmaven.javadoc.skip=true -B`, there will be the following Log.
```shell
[ERROR] Failed to execute goal on project shardingsphere-test-e2e-fixture: Could not resolve dependencies for project org.apache.shardingsphere:shardingsphere-test-e2e-fixture:jar:5.3.1-SNAPSHOT: org.apache.shardingsphere:shardingsphere-agent-bootstrap:jar:5.3.1-SNAPSHOT was not found in https://repository.apache.org/snapshots during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of apache.snapshots has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :shardingsphere-test-e2e-fixture
```
- It seems that `org.apache.shardingsphere:shardingsphere-agent-bootstrap` used by `org.apache.shardingsphere:shardingsphere-test-e2e` no longer exists. Should it be changed to `org.apache.shardingsphere:shardingsphere-agent-core`?
![image](https://user-images.githubusercontent.com/20187731/210327340-7ace42b9-7ba7-4210-b98e-ac776f28029e.png)

--- a/test/e2e/fixture/pom.xml
+++ b/test/e2e/fixture/pom.xml
@@ -39,7 +39,7 @@
     <dependencies>
         <dependency>
             <groupId>org.apache.shardingsphere</groupId>
-            <artifactId>shardingsphere-agent-bootstrap</artifactId>
+            <artifactId>shardingsphere-agent-core</artifactId>
             <version>${project.version}</version>
         </dependency>
         <dependency>

